db/schema.rb

```diff
@@ -46,6 +46,7 @@
     t.datetime "offer_deferred_at"
     t.string "status_before_deferral"
     t.datetime "reject_by_default_feedback_sent_at"
+    t.index ["application_form_id", "course_option_id"], name: "index_course_option_to_application_form_id", unique: true
```

Comment:
I don't think you want to be adding this change as part of this PR

Reply:
you're right sorry. Should've checked the schema 🤦